packages/convenient_test_manager/golden-test.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+HELPMSG="Run golden tests in an Xvfb container.
+Scenario: you want to test goldens at home or on CI.
+It might break, because you have a different screen size
+than the machine the golden was created on.
+This creates a virtual X server that will run the test.
+It has set screen size, so the goldens will always be deterministic
```

Comment:
Hmm IMHO when using the non-integration test, we can set arbitrary (fake) screen sizes, so it may be easier and stabler

Reply:
oh, got any docs on that?

Reply:
nvm I think I found it